.gitignore

```diff
@@ -1,2 +1,3 @@
+.idea
```

Comment:
please remove

Reply:
OK, it's me first Pull Request in GitHub. 
Can i make something more for Spanish Translation?
Great Library!